zitadel/terraform/terraform.tfvars

```diff
@@ -0,0 +1,5 @@
+jwt_profile_file = "local-token.json"
+redirect_xpanse-ui_uris    = ["http://localhost:3000/authentication/callback"]
+post_logout_redirect_uris   = ["http://localhost:3000","http://localhost:3000/credentials","http://localhost:3000/home","http://localhost:3000/register","http://localhost:3000/monitor","http://localhost:3000/","http://localhost:3000/catalog","http://localhost:3000/services","http://localhost:3000/myServices","http://localhost:3000/order","http://localhost:3000/createService"]
```

Comment:
@jinyangyang222 here we must store only the URIs.. ["", "/credentials", "/home], etc
The gui_base_url must be in the environment related var file.

Reply:
fixed